test/Autofac.Test/Features/LightweightAdapters/LightweightAdapterRegistrationExtensionsTests.cs

```diff
@@ -149,7 +147,7 @@ public DecoratorParameterization()
             [Fact]
             public void ParametersGoToTheDecoratedInstance()
             {
-                var expected = new Implementer1();
+                _ = new Implementer1();
```

Comment:
Uh... what is... uh... going on here?

Reply:
The refactoring suggested I could use the discard operator, because 'expected' isn't used, but what I should actually have done is just remove the line.